### ISSUE TITLE
CoreFx build process is copying test files into the original folder

### ISSUE THREAD

**Issue by saurabh500** [dotnet/corefx#15474]
The System.Data.SqlClient.ManualTests rely on the two files, data.xml and MultipleResultsTest.bsl being available in the binary folder. 
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/tests/ManualTests/System.Data.SqlClient.ManualTesting.Tests.csproj#L68
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/tests/ManualTests/System.Data.SqlClient.ManualTesting.Tests.csproj#L71

The two files used to get copied into the same folder as the Test DLL. Lately the files are being copied to a path which is the same as the original location of the files in the test source, i.e. netcoreapp\DDBasics\DDDataTypesTest\data.xml and netcoreapp\ProviderAgnostic\MultipleResultsTest\MultipleResultsTest.bsl 

This is causing the SqlClient tests, which run with the SqlServer, to break when run on dev machines. 

How do we get the files to copy to the same output location as before, without changing the test code?

**Comment by saurabh500** [comment]
cc @weshaggard @ericstj

**Comment by karelz** [comment]
cc @mellinoe

**Comment by weshaggard** [comment]
Nothing in our recent engineering changes should have impacted this. However if you want those files to be in the same directory as the test assembly then you can update the project to:

```
    <None Include="DDBasics\DDDataTypesTest\data.xml">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      <Link>data.xml</Link>
    </None>
    <None Include="ProviderAgnostic\MultipleResultsTest\MultipleResultsTest.bsl">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      <Link>MultipleResultsTest.bsl</Link>
    </None>
```

**Comment by mellinoe** [comment]
Yes, I was about to post as much. I put a PR here that fixes this: #15485
